backend/authentication/password_validators.py

```diff
@@ -24,3 +25,15 @@ def get_help_text(self):
             "Your password must contain at least 1 numeric character and 1 alphabetic "
             "character"
         )
+
+
+class PasswordStrengthValidator:
+    def validate(self, password, user=None, resultLabel, result):
```

Comment:
this should just take in `password` and optionally `user`. resultLabel and result are from inside the function, they shouldn't go here.
`def validate(self, password, user=None):`